### DISCUSSION TITLE
How To Hide A Row

### DISCUSSION THREAD

**Discussion by ducanhkatori00** [serenity-is/Serenity#5719] [Help]
Hello,
I have 3 permissions: Employee, HR, Admin
When status field = "Pending" I want to only show for Employee.
How to do it? Please help me. 
Thanks a lot!

![image](https://user-images.githubusercontent.com/77576309/118217725-40364e80-b4a0-11eb-9c52-0bab92f72010.png)

**Comment by minhhungit** [comment]
I think you can handle it by adding query condition in Repository/MyListHandler, something like this:

![image](https://user-images.githubusercontent.com/2279508/118221342-9e1a6480-b4a7-11eb-80ca-763449352104.png)


Search more using this keyword **mylisthandler** https://github.com/serenity-is/Serenity/search?q=mylisthandler&type=issues

**Reply by ducanhkatori00** [reply]
Omg. Thanks a lot! Can you give me your facebook? I wanna add friend with you.
p/s: e chào a ạ :D. Đây là facebook của e: https://www.facebook.com/09356887xx/. 
Rất mong được làm quen với a.
